docs/ruleset_functions/get_zones_computer_rooms.md

```diff
@@ -0,0 +1,27 @@
+## get_zones_computer_rooms
+
+**Description:** Get the list of zones that have computer rooms associated with them in the P-RMR.  
+
+**Inputs:**
+- **P-RMR**: To determine if any of the zones have computer rooms associated with them.
+
+**Returns:**
+- **applicable_computer_room_list_p**: A list that saves all zones that have computer rooms associated with them.
+ 
+**Function Call:** 
+
+
+**Logic:**
+- For each zone in P_RMR: `for zone_p in P_RMR...zones:`
+    - Reset applicability flag: `rule_applicability_check = FALSE` 
```

Comment:
small comment to rename rule_applicability_check to something like 'has_computer_room'

Reply:
Good point, modified.